book/src/keymap.md

```diff
@@ -166,6 +166,9 @@ These sub-modes are accessible from normal mode and typically switch back to nor
 | `Ctrl-w` | Enter [window mode](#window-mode)                  | N/A            |
 | `Space`  | Enter [space mode](#space-mode)                    | N/A            |
 
+These modes (except `:`) can be configured by
```

Comment:
minor nit but can we call this "command mode" instead of `:`? I think using `:` might make readers think that you can't remap the binding for the `command_mode` command to something other than `:` rather than the commands within command mode

```suggestion
These modes (except command mode) can be configured by
```

Reply:
yeah, that makes more sense